tests/noise/test_noisy_execution.py

```diff
@@ -0,0 +1,83 @@
+import sys
+
+import numpy as np
+import pytest
+
+from mpqp import QCircuit
+from mpqp.core.instruction.measurement import (
+    BasisMeasure,
+    ExpectationMeasure,
+    Observable,
+)
+from mpqp.execution import ATOSDevice, AvailableDevice, AWSDevice, run
+from mpqp.gates import *
+from mpqp.noise import Depolarizing
+
+
+@pytest.fixture
+def circuit():
+    return QCircuit(
+        [
+            H(0),
+            X(1),
+            Y(2),
+            Z(0),
+            S(1),
+            T(0),
+            Rx(1.2324, 2),
+            Ry(-2.43, 0),
+            Rz(1.04, 1),
+            Rk(-1, 0),
+            P(-323, 2),
+            U(1.2, 2.3, 3.4, 2),
+            SWAP(2, 1),
+            CNOT(0, 1),
+            CZ(1, 2),
+            # CRk(4, 2, 0)
+        ]
+    )
+
+
+@pytest.fixture
+def devices():
+    devices: list[AvailableDevice] = [AWSDevice.BRAKET_LOCAL_SIMULATOR]
+    if "-l" in sys.argv or "--long" in sys.argv:
```

Comment:
This works for this test, but no more for other tests in tests/execution/providers/atos.py for instance